### DISCUSSION TITLE
How to analyze the values of components in sub circuits

### DISCUSSION THREAD

**Discussion by pxmonti** [hneemann/Digital#746] [Q&A] [answered]
Hello all, I am creating an implementation of a R32I RISC-V processor with Digital. I have subcircuits for the 5 pipeline stages of the CPU, plus program memory and data memory. How can I see the state of the register file and the memory during simulation? I see that you cannot "open" the subcircuits during simulation. I would like to see the content of registers and memory.
Thank you.

**Comment by hneemann** [comment, marked as answer]
Open the measurement value table (F6).
There you will find all registers and memories used in the current circuit. 
The registers are displayed directly and the RAMs are available in the menu.

**Reply by pxmonti** [reply]
Thank you, it was there all the time and I didn't realize that!

**Reply by pottsali** [reply]
Oh, wow, I've been using Digital for a couple of years and I never realised I could do that. That makes life so much easier, thank you!
